grudge/trace_pair.py

```diff
@@ -344,38 +365,27 @@ def cross_rank_trace_pairs(
     components, respectively. Each of the TracePair components are structured
     like *ary*.
 
-    :arg ary: a single :class:`~meshmode.dof_array.DOFArray`, or an object
-        array of :class:`~meshmode.dof_array.DOFArray`\ s
-        of arbitrary shape.
+    During the case when *ary* is a number, rather than a
+    :class:`~meshmode.dof_array.DOFArray` or an
+    :class:`~arraycontext.container.ArrayContainer` of them, it is assumed
+    that the same number is being communicated on every rank.
```

Comment:
```suggestion
    If *ary* is a number, rather than a
    :class:`~meshmode.dof_array.DOFArray` or an
    :class:`~arraycontext.container.ArrayContainer` of them, it is assumed
    that the same number is being communicated on every rank.
```